express/kfp_utils.py

```diff
@@ -18,3 +28,56 @@ def parse_kfp_list(kfp_parsed_string: str) -> list:
         list: the list representation of the json string
     """
     return ast.literal_eval("".join(kfp_parsed_string))
+
+
+def configure_pipeline(file_path: str) -> None:
```

Comment:
Two comments on this function:

- It's not clear what it does based on the name. If it only sets the image pull policy, let's name the function accordingly.
- Do we actually need to do this? This is a bit magical, which may lead to unexpected behavior, and I don't see why we cannot just add this to all our `component.yaml` files explicitly?

Reply:
I'll remove it, this was originally copied over from some the KFP boilerplate but I guess we won't need it in our case